runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperConfig.java

```diff
@@ -70,11 +70,21 @@ default boolean quickhopOutOfDanger()
 		return true;
 	}
 
+	@ConfigItem(
+		keyName = "quickHopLimitRegion",
```

Comment:
This key name should match the method name

Reply:
Fixed now. Also removed an old import that I forgot to remove in a previous commit.